Form/form_us.xml

```diff
@@ -2403,7 +2403,7 @@
                    <_attribute>Months unemployed during census year</_attribute>
                </column>
                <column>
-                   <_attribute>(Day of the Enumerator’s visit) sick or temporarily disabled</_attribute>
+                   <_attribute>(Day of the Enumerator's visit) sick or temporarily disabled</_attribute>
```

Comment:
Kurt is the maintainer of the US form definitions, so he should make the decision.